pkg/runner/runner.go

```diff
@@ -60,15 +60,15 @@ func RunBundle(action string, ns string, bundleName string, sandboxRole string,
 	if len(candidateSpecs) == 0 {
 		if len(bundleRegistry) > 0 {
 			log.Errorf("Didn't find APB [%v] in registry [%v]\n", bundleName, bundleRegistry)
-			return
+			return "", errors.New(fmt.Sprintf("failed to find APB [%v] in registry [%v]", bundleName, bundleRegistry))
```

Comment:
I think we're going to end up with one too many error messages being printed here.

Reply:
```
ERR - Didn't find APB [bar] in registry foo
ERR - Failed to execute bundle [bar]: [failed to find APB [bar] in registry [foo]]
```

^ Above will be the output from what I can tell. Seems redundant to print info about not being able to find the APB multiple times.